ext/page_log/palm/palm_kv.h

```diff
@@ -38,6 +38,12 @@
 #define WT_MILLION 1000000
 #endif
 
+/*
+ * PALM flags start at the 16th bit (0x10000u) to avoid conflicts with __wt_page_log_put_args flags,
+ * starting at the next power of 2 after 0x8000 (32768).
+ */
+#define WT_PALM_KV_TOMBSTONE 0x10000u
```

Comment:
@mariammojid, should we start at `0x20000u`? It looks like `0x10000u` would be an accepted value for the PALI flags. I believe we don't exclude the stop boundary.